lib/matplotlib/cm.py

```diff
@@ -476,36 +474,10 @@ def autoscale_None(self):
         self.norm.autoscale_None(self._A)
         self.changed()
 
-    def _add_checker(self, checker):
-        """
-        Add an entry to a dictionary of boolean flags
-        that are set to True when the mappable is changed.
-        """
-        self._update_dict[checker] = False
-
-    def _check_update(self, checker):
-        """Return whether mappable has changed since the last check."""
-        if self._update_dict[checker]:
-            self._update_dict[checker] = False
-            return True
-        return False
-
     def changed(self):
         """
         Call this whenever the mappable is changed to notify all the
         callbackSM listeners to the 'changed' signal.
         """
         self.callbacksSM.process('changed', self)
-        for key in self._update_dict:
-            self._update_dict[key] = True
         self.stale = True
-
-    update_dict = _api.deprecate_privatize_attribute("3.3")
```

Comment:
Removing deprecated API requires a note in API changes.

Reply:
:+1: Added a note.